src/doc/documentation-guide.md

```diff
@@ -94,6 +95,68 @@ that concatenates the various `README.md` files we have.
 
 # Stylistic Conventions
 
+## General Considerations
+
+Documentation is most effective when it is precise, concise and uniform.
+
+ - **Preciseness**. Vague descriptions do more harm than good as different
```

Comment:
I don't think stating three words would be enough, and I think that explaining the concepts with reasoning and specific examples is what communicates the intent clearly.